src/lib/y2packager/dialogs/addon_selector.rb

```diff
@@ -63,13 +72,21 @@ def abort_handler
       #
       # @return [String]
       def help_text
-        # TRANSLATORS: help text (1/2)
+        # TRANSLATORS: help text
         _("<p>The selected repository contains several products in independent " \
-        "subdirectories. Select which products you want to install.</p>") +
-          # TRANSLATORS: help text (2/2)
-          _("<p>Note: If there are dependencies between the products you have " \
-          "to manually select the dependent products. The product dependencies "\
-          "cannot be automatically detected and checked.</p>")
+        "subdirectories. Select which products you want to install.</p>")
+      end
+
+      # Handle changing the current item or chaging the selection
+      def addon_repos_handler
+        current_item = Yast::UI.QueryWidget(Id(:addon_repos), :CurrentItem)
+        current_product = products.find { |p| p.dir == current_item }
+
+        return unless current_product
+
+        refresh_details(current_product)
+
+        select_dependant_products
```

Comment:
well, spelling of dependant catch my eyes, so I check it and looks like according to british english it should be used only as noun. In american it does not matter https://grammarist.com/spelling/dependant-dependent/